CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
-# Contributing to Magistrala
+# Contributing to SuperMQ
 
-The following is a set of guidelines to contribute to Magistrala and its libraries, which are
+The following is a set of guidelines to contribute to SuperMQ and its libraries, which are
 hosted on the [Abstract Machines Organization](https://github.com/absmach) on GitHub.
 
 This project adheres to the [Contributor Covenant 1.2](http://contributor-covenant.org/version/1/2/0).
```

Comment:
I think we don't have domain magistrala.com, if it is true, then we need to rename `@magistrala.com` to `@abstractmachines.fr`
I